uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/ILabTest.java

```diff
@@ -34,14 +35,14 @@ public interface ILabTest {
     /**
      * Get the query that can be used to pull a user from the LAB API.
      *
-     * @return A {@link LabUserQuery} object that can be used to pull user via LAB API
+     * @return A {@link LabQuery} object that can be used to pull user via LAB API
      */
-    LabUserQuery getLabUserQuery();
+    LabQuery getLabQuery();
 
     /**
      * Get the type of temp user that can be used to create a new temp user via LAB API.
      *
-     * @return The type of temp user as denoted in {@link com.microsoft.identity.internal.testutils.labutils.LabConstants.TempUserType}
+     * @return The type of temp user as denoted in {@link TempUserType}
      */
     String getTempUserType();
```

Comment:
Let's refactor this method to return the `TempUserType` enum constant directly. That way then you don't have to invoke `.name()` on the constant during the tests